src/main/java/cz/fi/muni/pa165/esports/entity/Player.java

```diff
@@ -21,7 +26,7 @@
     @ManyToOne
     private Team team;
 
-    private Integer age;
+    private Integer year;
```

Comment:
Maybe add javadoc saying `year` represents year of birth? (also to methods working with them)